pom.xml

```diff
@@ -112,6 +112,26 @@
                 </dependency>
             </dependencies>
         </profile>
+        <profile>
+            <id>yugabyte</id>
+            <properties>
+                <classifier>yugabyte</classifier>
+            </properties>
+            <dependencies>
+                <dependency>
+                    <groupId>com.yugabyte</groupId>
+                    <artifactId>jdbc-yugabytedb</artifactId>
+                    <version>42.3.0</version>
```

Comment:
Please use the latest version [42.3.4](https://mvnrepository.com/artifact/com.yugabyte/jdbc-yugabytedb/42.3.4)